hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/NearCachedClientCacheProxy.java

```diff
@@ -146,7 +146,7 @@ protected ClientDelegatingFuture getAsyncInternal(Data dataKey, ExpiryPolicy exp
         try {
             long reservationId = nearCache.tryReserveForUpdate(dataKey);
             ClientDelegatingFuture future = super.getAsyncInternal(dataKey, expiryPolicy, callback);
-            future.andThenInternal(new GetAsyncCallback(dataKey, reservationId, callback), true);
+            future.andThenInternal(new GetAsyncCallback(dataKey, reservationId), true);
```

Comment:
This is the fix: the statistics maintenance callback is already registered with `andThenInternal` in above statement, no need to execute it again within `GetAsyncCallback`.